logprep/run_logprep.py

```diff
@@ -25,6 +24,31 @@
 from logprep.util.schema_and_rule_checker import SchemaAndRuleChecker
 from logprep.util.time_measurement import TimeMeasurement
 
+from logging import (
+    getLogger,
+    CRITICAL,
+    FATAL,
+    ERROR,
+    WARNING,
+    INFO,
+    DEBUG,
+    NOTSET,
+    basicConfig,
+    Logger,
+)
+from logging.handlers import SysLogHandler
+
+name_to_level = {
+    "CRITICAL": CRITICAL,
+    "FATAL": FATAL,
+    "ERROR": ERROR,
+    "WARN": WARNING,
+    "WARNING": WARNING,
+    "INFO": INFO,
+    "DEBUG": DEBUG,
+    "NOTSET": NOTSET,
+}
```

Comment:
you don't need this, as you can throw in the logel-level string from `log_config.get("level", ...)` directly into the `basicConfig(level=log_level, ...)` call.
```suggestion
```